packages/onboarding/src/step-container-v2/wireframes/PlaygroundLayout/PlaygroundLayout.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from 'clsx';
+import { Content } from '../../components/Content/Content';
+import { StepContainerV2 } from '../../components/StepContainerV2/StepContainerV2';
+import { ContentProp } from '../../components/StepContainerV2/context';
+import { TopBarRenderer } from '../../components/TopBar/TopBarRenderer';
+
+import './style.scss';
+
+interface PlaygroundLayoutProps {
+	topBar?: ContentProp;
+	className?: string;
+	children?: ContentProp;
+}
+
+export const PlaygroundLayout = ( { topBar, className, children }: PlaygroundLayoutProps ) => {
```

Comment:
@zaerl I made a special purpose wireframe just for the playground step. It was using the `hasContentPadding` prop to bypass the standard stepper layout patterns. However this prop is really there only as a hack for the Checkout page, we don't want it to become the norm to use this prop.

The playground step diverges so far from the existing stepper patterns that I actually think it warrants a separate wireframe. @zaerl I don't know if this is the final design or whether you're just prototyping? If the playground step ends up looking different (maybe more like the design preview step?) then you might want to switch it back to one of the regular wireframes down the road.

You can see the playground step by navigating to `/setup/onboarding/playground` locally.

Reply:
![CleanShot 2025-04-01 at 17 47 43@2x](https://github.com/user-attachments/assets/85afb8db-d8c4-4ed1-b5b5-885d25970aaf)

Reply:
Ok, sounds like a custom wireframe is best then 👍